torch_frame/data/dataset.py

```diff
@@ -585,19 +616,6 @@ def _get_tensorframe_converter(self) -> DataFrameToTensorFrameConverter:
             col_to_time_format=self.col_to_time_format,
         )
 
-    def _update_col_stats(self):
-        r"""Set :obj:`col_stats` based on :obj:`tensor_frame`."""
-        if torch_frame.text_embedded in self._tensor_frame.feat_dict:
-            # Text embedding dimensionality is only available after the tensor
-            # frame actually gets created, so we compute col_stats here.
-            offset = self._tensor_frame.feat_dict[
-                torch_frame.text_embedded].offset
-            emb_dim_list = offset[1:] - offset[:-1]
-            for i, col_name in enumerate(self._tensor_frame.col_names_dict[
-                    torch_frame.text_embedded]):
-                self._col_stats[col_name][StatType.EMB_DIM] = int(
-                    emb_dim_list[i])
```

Comment:
I hope update col stats happen in `Dataset` side, rather than on the converter side.

Reply:
cool